test/helpers/test/advanceToBlock.test.js

```diff
@@ -0,0 +1,19 @@
+const advanceToBlock = require('../advanceToBlock');
+
+const BigNumber = web3.BigNumber;
+require('chai')
+  .use(require('chai-bignumber')(BigNumber))
+  .should();
+
+describe('advanceToBlock', function () {
+  beforeEach(function () {
+    this.startingBlock = web3.eth.blockNumber;
+  });
+
+  describe('advanceBlock', function () {
+    it('increases the block number by one', async function () {
+      await advanceToBlock.advanceBlock();
```

Comment:
What about rename `advanceToBlock` to something like `blocks`? or `miner`?

Reply:
I also don't like this, but users will usually import the function directly (i.e. `const { advanceToBlock} = require('advanceBlock')`). I imported the whole module in the test to make it clear what each function is testing.

These should all be replaced once we mock all time usage anyway.